terraform/modules/ops/build.tf

```diff
@@ -27,10 +28,11 @@ resource "google_cloudbuild_trigger" "api_push_to_main" {
 }
 
 resource "google_cloudbuild_trigger" "web_push_to_main" {
-  project  = var.project_id
-  count    = var.setup_cd_system ? 1 : 0
-  name     = "web-push-to-main"
-  filename = "ops/web-build.cloudbuild.yaml"
+  project     = var.project_id
+  count       = var.setup_cd_system ? 1 : 0
+  name        = "web-new-build"
```

Comment:
This name change goes from `<short component>-<triggering action>` to `<short component>-<build objective>`.  This seems good to me if we don't plan to expand scope of this trigger's goals.

I wonder if "new" really adds value over "web-build" but will follow your judgement.